pkg/volume/flexvolume/attacher-defaults.go

```diff
@@ -38,8 +38,12 @@ func (a *attacherDefaults) WaitForAttach(spec *volume.Spec, timeout time.Duratio
 
 // GetDeviceMountPath is part of the volume.Attacher interface
 func (a *attacherDefaults) GetDeviceMountPath(spec *volume.Spec) string {
-	name := a.driverName
-	return a.host.GetPodVolumeDir(spec.PodUID, utilstrings.EscapeQualifiedNameForDisk(name), spec.Name())
+	driverDir := path.Join(a.host.GetPluginDir(flexVolumePluginName), a.driverName)
+
+	if spec.PersistentVolume != nil {
+		return path.Join(driverDir, "pvs", spec.PersistentVolume.Name)
+	}
+	return path.Join(driverDir, "pods", string(spec.PodUID), spec.Volume.Name)
```

Comment:
we need a global name independent of pod.. that way the filesystem can be bind mounted in other pods.